web/src/components/layout/Layout.jsx

```diff
@@ -31,10 +32,20 @@ const FooterActions = createTeleporter();
 const FooterInfoArea = createTeleporter();
 
 /**
- * D-Installer main layout component.
  *
- * It displays the content in a single vertical responsive column with fixed
- * header and footer.
+ * The D-Installer main layout component.
+ *
+ * It displays the content in a single column with a fixed header and footer.
+ *
+ * To achieve an {@link https://gregberge.com/blog/react-scalable-layout scalable layout},
+ * it uses {@link https://reactjs.org/docs/portals.html React Portals} through
+ * {@link https://github.com/gregberge/react-teleporter react-teleporter}. In other words,
+ * it is mounted only once and gets influenced by other components by using the created
+ * slots (Title, PageIcon, MainActions, etc).
+ *
+ * So, please ensure that {@link test-utils!mockLayout } gets updated when adding or deleting
+ * slots here. It's needed in order to allow testing the output of components that interact
+ * with the layout using that mechanism.
```

Comment:
Updated here due to changes done in the testing side. It adds comments missing at https://github.com/yast/d-installer/pull/216